packages/core/src/common/dom/set-unique-element-id.ts

```diff
@@ -13,7 +13,10 @@ export default function setUniqueElementId(selector: string, idPrefix: string):
 
     return Array.prototype.slice.call(containers).map((container: HTMLElement) => {
         if (!container.id) {
-            container.id = uniqueId(idPrefix);
+            const getUniqId = (id: string): string =>
+                document.getElementById(id) ? getUniqId(uniqueId(idPrefix)) : id;
```

Comment:
Can we declare this outside `setUniqueElementId` method please?

Reply:
Also seems does it make sense to add the check in `initializeButton` method in checkout-button-initializer.

Reply:
> Can we declare this outside `setUniqueElementId` method please?

Would it be ok if I move it to the same file before the `setUniqueElementId` is declared? Or create another file for this?

Reply:
> Also seems does it make sense to add the check in `initializeButton` method in checkout-button-initializer.

Check is already added by using `setUniqueElementId` in `initializeButton`

Reply:
@animesh1987 what do you think?